bindings/matlab/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+find_program(
+	MATLAB_EXECUTABLE
+	NAMES matlab
+	PATHS ${Matlab_ROOT_DIR} ${Matlab_ROOT_DIR}/bin
```

Comment:
I should note we could bundle the FindMatlab module and just use that, but I don't think we need all the extra functionality that provides at this point.